### PR TITLE
[MD1-3702] Landing page

### PR DESCRIPTION
**Pull request by Florian-Eschenbacher**
## Description

- the new static landing page with the old wizard
- trustpilot is working now
- google tag manager was added

## DoD checklist

- [ ] I have performed a self-review of my own code
- [ ] I added integration tests for the feature
- [ ] I added tracking events
- [ ] I tested the feature on Internet Explorer 11
- [ ] I tested the feature on my phone
- [ ] QA has approved this feature
- [ ] I added the ⚠️ flag if this feature requires a backend task before merging

## Links to required backend PRs

- link

## Jira Ticket

https://meindach.atlassian.net/browse/MD1-3702

## Feature URL

https://pr-6.wizard-interview.ui.meindach.de

### CODE REVIEW

File: src/app/components/about.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.about {
+  margin: 0 0 180px 0;
+  @include stylesFor(mobile) {
+    margin-bottom: 100px;
+  }
+}
+
+.about__section__title {
```
Comment (jruz) [inline src/app/components/about.scss]: you can write this in a sassy way
```scss
.about {
  ...
  &__section__title {
    ...
  }
}
```

Comment (Florian-Eschenbacher) [reply src/app/components/about.scss]: Yep thats a leftover from a more verbose previous version of BEM syntax.

File: src/app/components/service.cljs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+(ns app.components.service)
+
+(defn service []
+  [:section.service
+   [:h1.service__title "Wir sind für Sie da"]
+   [:p.service__paragraph "Sie haben Fragen zu MeinDach und möchten gene von einem unserer Servicemitarbeiter mehr erfahren? Rufen Sie uns jederzeit an."]
+   [:a.service__phone {:href "tel:061 04 937 100"} "061 04 937 100"]
```
Comment (jruz) [inline src/app/components/service.cljs]: maybe can be a helper

File: src/app/components/wizardcontainer.cljs [hunk 3]
```diff
@@ -2,16 +2,5 @@
...
-     :height "85"}]
-   [:div.wizardcontainer__root]
-   [:img.wizardcontainer__houses
-    {:alt "roofs"
-     :src
-     "https://images.ctfassets.net/qxxnae47u69u/19Oe5hl1G4Uq0axZJGzjH8/0af222dd7728f49de42e251dbd1644b4/IMG01.jpg"
-     :width "519"
-     :height "786"}]])
\ No newline at end of file
+   [:div.wizardcontainer__container
```
Comment (jruz) [inline src/app/components/wizardcontainer.cljs]: please use only one div for this

File: src/styles/basics/mixins.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  text-decoration: none;
+  color: var(--color-text-default);
+  &::before {
+    content: '   ';
+    display: inline-block;
+    width: 22px;
+    height: 22px;
+    margin-right: 5px;
+    margin-bottom: -7px;
+    background: url('https://images.ctfassets.net/qxxnae47u69u/gfN608RfwLK13T7f8kv4y/aac31c6f664cafd6de26852d7ce95ab5/headset-yellow.svg')
```
Comment (jruz) [inline src/styles/basics/mixins.scss]: please use backgound-image and backgroud-position so its clearer, scss is compiled there's no need to pre minimize it :)

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,4 +1,16 @@
 @import './basics/basics.scss';
 @import './basics/fonts.scss';
-@import './components/wizardcontainer.scss';
-@import './components/footer.scss';
+@import './basics/breakpoints.scss';
+@import './basics/colors.scss';
+@import './basics/mixins.scss';
+
```
Comment (jruz) [inline src/styles/index.scss]: how about we move this into app and there create a folder styles

**Pull request merged**